tests/test_decompiler.py

```diff
@@ -1752,6 +1779,10 @@ def test_decompiling_tac_base_len(self, decompiler_options=None):
 
     @for_all_structuring_algos
     def test_decompiling_dd_argmatch_to_argument_noeagerreturns(self, decompiler_options=None):
+        if decompiler_options is None:
```

Comment:
I don't like seeing this code many times. Look at my other comment on the function you introduced for how you can do this.